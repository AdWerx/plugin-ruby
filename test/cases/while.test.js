runCase("while.rb");
runCase("while.rb", { inlineLoops: false }, "block-loops.yml");
