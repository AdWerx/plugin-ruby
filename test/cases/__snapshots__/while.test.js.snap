// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`while.rb matches expected output 1`] = `
"# frozen_string_literal: true

1 while a

1 while a

1 while super_super_super_super_super_super_super_super_super_super_super_long

while super_super_super_super_super_super_super_super_super_super_super_supelong
  1
end

1 until a

1 until a

1 until super_super_super_super_super_super_super_super_super_super_super_long

until super_super_super_super_super_super_super_super_super_super_super_supelong
  1
end
"
`;
